DXRMissions/DeusEx/Classes/DXRMissionsParis.uc

```diff
@@ -334,15 +335,15 @@ function AfterMoveGoalToLocation(Goal g, GoalLocation Loc)
     if (Loc.name=="Media Store"){
         if ((g.name=="Nicolette") || (g.name=="Jaime" && dxr.flagbase.GetBool('JaimeLeftBehind'))){
             //Spawn a small spotlight just to make it a bit easier to spot them through the window
-            dl = DynamicLight(Spawnm(class'DynamicLight',,,vect(990,1768,300),rot(-16385,0,0)));
+            dl = Spawn(class'DynamicLight',,,g.actors[0].a.Location+vectm(-16,0,113),rot(-16385,0,0));
```

Comment:
does this need rotm too?

```suggestion
            dl = Spawn(class'DynamicLight',,,g.actors[0].a.Location+vectm(-16,0,113),rotm(-16385,0,0));
```

Reply:
This is just pitch, so I don't think it's strictly necessary (with our current map rotations), but can't hurt

Reply:
oh yea it's just a straight down light so Yaw doesn't matter